tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/Preview/Display Marketing Page Preview Client Application Types Section.feature

```diff
@@ -11,7 +11,6 @@ Background:
     And Solutions exist
         | SolutionId | SolutionName   | SupplierStatusId | SupplierId |
         | Sln1       | MedicOnline    | 1                | Sup 1      |
-        | Sln2       | TakeTheRedPill | 1                | Sup 2      |
         | Sln3       | PracticeMgr    | 1                | Sup 2      |
         | Sln4       | PracticeMgr    | 1                | Sup 2      |
         | Sln5       | Potions        | 1                | Sup 1      |
```

Comment:
I think it would be better for the solution IDs to be consecutive.

Reply:
Updated solution Ids in consecutive order